portal/server/utils/csv-utils.test.js

```diff
@@ -21,6 +22,20 @@ describe('csv-utils', () => {
     });
   });
 
+  describe('excelColumnToColumnIndex', () => {
+    it('returns the correct index for a column below 26', async () => {
+      const columnIndex = excelColumnToColumnIndex('C');
+
+      expect(columnIndex).toBe(2);
+    });
+
+    it('returns the correct index for a column above 26', async () => {
+      const columnIndex = excelColumnToColumnIndex('AE');
+
+      expect(columnIndex).toBe(30);
+    });
```

Comment:
What's the significance of 26 here?

Reply:
This just represents wrapping from a single character column code to multiple characters. I can expand the test case name to include this if that's helpful?

Reply:
Ah, of course. Probably wouldn't hurt to be a bit more explicit